client/tx/tx.go

```diff
@@ -23,6 +23,23 @@ import (
 	authsigning "github.com/cosmos/cosmos-sdk/x/auth/signing"
 )
 
+func ValidateMsg(msg sdk.Msg) error {
```

Comment:
why do we need this? I thought all Msg's ValidateBasic already checks that the signers are correct bech32 addresses.

Reply:
I think it may not cover signer addresses in few cases, can you check [this](https://github.com/cosmos/cosmos-sdk/issues/9239#issuecomment-839935107)

Reply:
Can we have a godoc for ValidateMsg?